File: preparation/tinkoff_contests/contest_C/F.py
n = int(input())
arr = []
for _ in range(n):
    num = int(input())
    arr.append(num)

permutations = set()
for i in range(0,len(arr)-2):
    if (arr[i] > arr[i+1] > arr[i+2]): 
        permutations.add((arr[i], arr[i+1], arr[i+2]))

print(permutations)

File: ~learning/informatics_course/1_searching&sorting/1_problems/F.cpp
#include <iostream>
#include <sstream>
#include <climits>
#include <vector>
#include <string>
using namespace std;

int main(){
    int n;
    cin >> n;
    cin.ignore();

    string input;
    vector<int> arr;
    getline(cin, input);
    stringstream ss(input);
    int i;
    while (ss >> i) arr.push_back(i);

    int maxI;
    int maxV = INT_MIN;
    for (int i = 0; i < n; i++){
        if (arr[i] > maxV){
            maxV = arr[i];
            maxI = i;
        }
    }

    cout << maxI+1;

    return 0;
}